### PR TITLE
Holopictures and more!

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: 

:cl: Dax Dupont
add: You can now insert holodisks into cameras and take a static holographic picture of someone!
add: Hologram recordings can now be offset slightly.
/:cl:

[why]: Holograms are neat and now you can take holopictures of others too!

### CODE REVIEW

**Comment by ayyourboibilly** [comment]
There should be a holo imager for detective that shows what happend during the crime

File: code/modules/paperwork/photography.dm [hunk 1]
```diff
@@ -437,13 +454,24 @@
 /obj/item/device/camera/afterattack(atom/target, mob/user, flag)
 	if(!on || !pictures_left || !isturf(target.loc))
 		return
-
-	captureimage(target, user, flag)
+	if (disk)
+		if(ishuman(target))
```
Comment (RemieRichards) [inline code/modules/paperwork/photography.dm]: what if I want to be called by a monkey? or a potato?

**Comment by DaxDupont** [comment]
Called? This is a camera.

**Comment by RemieRichards** [comment]
@DaxDupont eh, not 100% relevant but still, my point is it's an arbitrary restriction.

**Comment by RemieRichards** [comment]
or a potato!

**Comment by DaxDupont** [comment]
I don't want to add objects to it really.

**Comment by vuonojenmustaturska** [comment]
sure, why not

**Pull request merged**